lib/multiply.rb

```diff
@@ -1,2 +1,9 @@
 def multiply(num1, num2)
-end
\ No newline at end of file
+output = []
+num1.times do
+    output << "#{num2}"
```

Comment:
not only does it not need to be a string, I dont want it to be, I was trying so hard trying to figure out how to take the value out of a string I didnt even think about the fact that it wasnt originally and my stupid self put it in one!  Thanks Sumeet!! -Michael Meier 

    On Tuesday, January 19, 2016 1:40 PM, Sumeet Jain <notifications@github.com> wrote:
 

 In lib/multiply.rb:> \ No newline at end of file
> +output = []
> +num1.times do
> +    output << "#{num2}"
"#{num2}" makes num2 a string. Does it need to be a string?Did you consider output << num2 as is?—
Reply to this email directly or view it on GitHub.